specs/ngsi-ld_faq.md

```diff
@@ -0,0 +1,170 @@
+# NGSI-LD FAQ
+
+## Introduction
+
+This FAQ compilation is intended to clarify NGSI-LD specification by providing
+answers to common questions.
+
+-   Q: What are the main (essential) differences between NGSI v2 and NGSI-LD?
+
+    -   R: The following:
```

Comment:
Remove indent and bullet ,

The bullet list below should be outdented one level (but the bullets remain)